src/common/target.c

```diff
@@ -168,7 +168,6 @@ static const TargetEntry TargetMap[] = {
     {   "sim6502",      TGT_SIM6502     },
     {   "sim65c02",     TGT_SIM65C02    },
     {   "supervision",  TGT_SUPERVISION },
-    {   "vc20",         TGT_VIC20       },
```

Comment:
Why remove "VC20"? That was the name of the VIC20 in Germany.
While I would not have introduced that alias at the beginning, I do not think it is useful to remove it now and possibly break some build environments. What do we gain from this removal?